src/frontend/src/pages/home/home.tsx

```diff
@@ -1,7 +1,94 @@
 import React from 'react';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import {
+    Link,
+} from "react-router-dom";
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Box from '@material-ui/core/Box';
+
 
 export const Home: React.FC = () => {
     return (
-        <div>Home Page</div>
+        <div>
+            <AppBar position="static">
+                <Toolbar>
+                    <IconButton edge="start" color="inherit" aria-label="menu">
+                        <AccountCircle />
+                    </IconButton>
+                    <Typography variant="h6" style={{ textAlign: "right" }} >
+                        0
+                    </Typography>
+                    <Link to="/login">
+                        <Button color="inherit">Logout</Button>
+                    </Link>
+                </Toolbar>
+            </AppBar>
+            <Grid
+                container
+                direction="column"
+                justify="center"
+                alignItems="center"
+                spacing={3}
+            >
+
+                <Grid item style={{ textAlign: "center" }}>
+                    <Link to="/new-challenge">
+                        <Button variant="contained" color="primary">
+                            New Challenge
+                    </Button>
+                    </Link>
+                </Grid>
+
+                <Grid item style={{ textAlign: "center" }}>
+                    <Typography variant="h6" component="h1" align="center">
+                        Challenges from friends
+                </Typography>
+                </Grid>
+
+
+                <Box display="flex" p={4} bgcolor="text.disabled">
```

Comment:
I would just surround this with a ```<Paper>``` element instead. Also, it should be surrounded by a ```<Grid item>``` component since it's still within your vertical ```<Grid container>```.
```suggestion
                <Grid item>
                    <Paper elevation={3}>
```